common/source/docs/common-lua-scripts.rst

```diff
@@ -532,6 +532,7 @@ To give Lua scripts access to more features of ArduPilot the API can be extended
 
 - Find the method or function you would like to expose to Lua. For example if you wanted to expose an additional feature of AHRS you would first find the method within `libraries/AP_AHRS/AP_AHRS.h <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_AHRS/AP_AHRS.h>`__. This can be an already existing method (function) or a method (function) newly added to the code.
 - Edit the `libraries/AP_Scripting/generator/description/bindings.desc <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/generator/description/bindings.desc>`__ and add a new line in the appropriate section for the method, or add a new section if a new class shall be added by following the examples of the other sections.
+- Add the method or function to `libraries/AP_Scripting/docs/docs.lua so that your IDE can provide coding prompts such as Intellisense in VSCode
```

Comment:
```suggestion
- Add the method or function to `libraries/AP_Scripting/docs/docs.lua<https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/docs/docs.lua>`__.
```
That should make it a proper link. I would remove the Intellisencse bit, its this extension. https://marketplace.visualstudio.com/items?itemName=sumneko.lua but I don't think its necessary to say what the docs are for. There is another section on how to setup the extension.